library/leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -252,6 +252,15 @@
       excluded.staticField("android.media.AudioManager", "mContext_static");
     }
 
+    if (SDK_INT > ICE_CREAM_SANDWICH && SDK_INT <= LOLLIPOP_MR1) {
+      // ActivityChooserModel holds a static reference to the last set ActivityChooserModelPolicy
+      // which can be a activity context.
+      // Observed here : https://code.google.com/p/android/issues/detail?id=172659
+      // Hack : https://gist.github.com/andaag/b05ab66ed0f06167d6e0
+      excluded.staticField("android.support.v7.internal.widget.ActivityChooserModel", "mActivityChoserModelPolicy");
```

Comment:
That one is interesting. It's not related to ICS / LOLLIPOP. It won't be fixed in AOSP, it has to be fixed in the support lib. So technically, it should be in an "if support lib < X" where X is the current latest. But we don't depend on the support lib, so I don't think we can. It's fine I guess..